### PR TITLE
[ADD] stock_default_warehouse_by_company module

### PR DESCRIPTION
**Pull request by chafique-delli**

### CODE REVIEW

File: stock_default_warehouse_by_company/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# © 2017 Chafique DELLI @ Akretion <chafique.delli@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': "Stock Default Warehouse by Company",
+    'summary': 'Add in the company, default warehouse for outgoing shipments '
+               'and incoming shipments',
+    'author': "Akretion",
+    'website': "http://www.akretion.com",
+    'category': 'Tools',
```
Comment (mourad-ehm) [inline stock_default_warehouse_by_company/__openerp__.py]: s/'category': 'Tools'/'category': 'Warehouse'/g

File: stock_default_warehouse_by_company/models/sale_stock.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+from openerp import models, api, fields
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    @api.model
+    def _get_default_warehouse(self):
+        company_id = self.env['res.users']._get_company()
+        company = self.env['res.company'].browse(company_id)
```
Comment (mourad-ehm) [inline stock_default_warehouse_by_company/models/sale_stock.py]: use env to get company company = self.env.user.company_id (for performance).

File: stock_default_warehouse_by_company/__openerp__.py (hunk 1 above)
Comment (chafique-delli) [inline stock_default_warehouse_by_company/__openerp__.py]: @mourad-ehm , it's fixed.

File: stock_default_warehouse_by_company/models/sale_stock.py (hunk 2 above)
Comment (chafique-delli) [inline stock_default_warehouse_by_company/models/sale_stock.py]: @mourad-ehm , It's fixed.
Thank you for the review.

File: stock_default_warehouse_by_company/models/company.py [hunk 3]
```diff
@@ -8,9 +8,9 @@
 class ResCompany(models.Model):
     _inherit = 'res.company'
 
-    out_warehouse_id = fields.Many2one(
+    delivery_warehouse_id = fields.Many2one(
         'stock.warehouse', string='Delivery Warehouse',
         help="Default warehouse to use on Sale Orders")
-    in_warehouse_id = fields.Many2one(
+    reception_warehouse_id = fields.Many2one(
```
Comment (bealdav) [inline stock_default_warehouse_by_company/models/company.py]: reception ? in english, really ?

Comment (chafique-delli) [inline stock_default_warehouse_by_company/models/company.py]: @david, Yes, but If you have another proposal, I am ok.

Comment (chafique-delli) [inline stock_default_warehouse_by_company/models/company.py]: @bealdav , Tu pensais à 'receipt_warehouse_id' ?

Comment (chafique-delli) [inline stock_default_warehouse_by_company/models/company.py]: @bealdav , it's fixed.

**Pull request merged**